### PR TITLE
KPI 14.3 - Council Reporting

### PR DESCRIPTION
**Pull request by mochet**
This PR adds the council report for round #18 (KPI 14.3)

### CODE REVIEW

**Review by bwhm — CHANGES_REQUESTED** [review]
I won't comment on the specifics of the "template" here, as I think I want to make that part of the new KPI scheme.

I do believe we that the `14.3` is incorrect though?

**Comment by bwhm** [comment]
That sounded a little harsh :)

What I meant was:
- The information looks good, and covers the basics
- Well written and organized
- The Council round number (18) is probably correct, but confusing (due to Rome != Constantinople)
- The KPI number is wrong either here, or in "our" blog :)

TL;DR
I will approve, but just want to clarify which is the "right" one!

**Comment by mochet** [comment]
> That sounded a little harsh :)
> 
> What I meant was:
> 
> * The information looks good, and covers the basics
> * Well written and organized
> * The Council round number (18) is probably correct, but confusing (due to Rome != Constantinople)
> * The KPI number is wrong either here, or in "our" blog :)
> 
> TL;DR
> I will approve, but just want to clarify which is the "right" one!

Thanks, there's a lot of improvements that can be done with the template, to make it a lot more readable / informative. I think it should start to incorporate a bit more data and a few things like an overview of each actor's participation level, a breakdown of vote types and a few things like that, but I'll wait to see the new KPI scheme and can improve it based upon that.

**Comment by bwhm** [comment]
I took the liberty of renaming the title :)

**Comment by mochet** [comment]
this has been approved via council vote: https://testnet.joystream.org/#/proposals/64

**Comment by blrhc** [comment]
> this has been approved via council vote: https://testnet.joystream.org/#/proposals/64

Great, merging!

**Review by blrhc — APPROVED** [review]
